### DISCUSSION TITLE
Storage location of database files

### DISCUSSION THREAD

**Discussion by c2006878** [nocodb/nocodb#6967] [General]
![1699925004756](https://github.com/nocodb/nocodb/assets/58558960/33cf7d8c-03d9-4b51-a955-86b5142aaca8)
I would like to know where the default connected database is located. If I upload a document, where should I go to find this document?

Additionally, I have connected to an external database and found the relevant path information【'1', '111', '2023-11-03 15:00:12', '2023-11-03 15:00:15', '[{"title":"xxxx.doc","mimetype":"application/msword","size":66088,"path":"download/noco/pc86i3cahrx9l3c/mtens4mpn1ohqu3/csol8s27xrcefpf/8k1-nu5GNkgFd4pp4m.doc","signedPath":"dltemp/9xfLfkPur0JHmr1u/1699002600000/noco/pc86i3cahrx9l3c/mtens4mpn1ohqu3/csol8s27xrcefpf/8k1-nu5GNkgFd4pp4m.doc"}]'
】about the document in the database tool after uploading the document. Where should I look for it now? I am currently conducting code tests locally, docker deployment has not been used yet.

**Comment by c2006878** [comment]
If deployed using Docker, is the data mounted to the path /usr/app/data/?

**Comment by wingkwong** [comment]
@c2006878 Yes.
